source/Calamari/Deployment/SpecialVariables.cs

```diff
@@ -106,6 +112,8 @@ public static class Features
         public static class Action
         {
             public const string Name = "Octopus.Action.Name";
+            public const string SkipRemainingConventions = "Octopus.Action.SkipRemainingConventions";
```

Comment:
Actually on second thought, `SkipRemainingConventions` is sometimes set by PowerShell scripts by users - e.g., in their PreDeploy.ps1, they might decide conditionally not to proceed with the package deployment, in which case they would set this variable. `SkipJournal` is new, but a similar argument could be made for it (if they conditionally decide not to install, they might also decide not to have it counted towards the previous install).